Dockerfile.toolkit

```diff
@@ -0,0 +1,35 @@
+# Elemental CLI is built in another Dockerfile that support multi-arch
```

Comment:
I'm not sure about this Dockerfile split, from what I can tell it can be a single Dockerfile that uses --target to split out which image to build.